embc-app/ClientApp/src/app/evacuee-registration/evacuee-registration.component.ts

```diff
@@ -86,38 +137,39 @@ export class EvacueeRegistrationComponent implements OnInit {
       return null;
     }
   }
+
   initForm(): void {
     this.form = this.formBuilder.group({
-      restrictedAccess: null,
+      restrictedAccess: true,
```

Comment:
Why setting this to **true** by default, should be left null so the volunteer makes a choice

Reply:
This was for debugging the form. I know that I had an error dominating my browser console regarding a missing value for this.